docs/introduction.rst

```diff
@@ -1,10 +1,22 @@
 Introduction to Flatpak
 =======================
 
-Flatpak is a framework for distributing desktop applications on Linux. It
-has been created by developers who have a long history of working on the
+Flatpak is a framework for distributing desktop applications across various Linux
+distributions. It has been created by developers who have a long history of working on the
 Linux desktop, and is run as an independent open source project.
 
+Terminology
+-----------
+
+- Flatpak: The Flatpak package manager.
```

Comment:
I don't think this is accurate. First, Flatpak isn't just a package manager, and I don't think that "package management" is its primary function or goal. Secondly, using the term "package manager" confuses it with systems like apt and dnf.